Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -120,7 +128,7 @@ block FirstOrderAMIGO
     "Check if an autotuning is ongoing while a new autotuning request is received"
     annotation (Placement(transformation(extent={{120,-72},{140,-52}})));
   Buildings.Controls.OBC.CDL.Utilities.Assert assMes(
-    message="An autotuning is ongoing and an autotuning request is ignored.")
+    message="*** Warning in " + getInstanceName() + "An autotuning is ongoing and an autotuning request is ignored.")
```

Comment:
We should not use the Modelica function `getInstanceName()` in OBC package. Please change the warning message as:
`Warning: An autotuning is ongoing and the new autotuning request is ignored.`

Reply:
fixed